ms-frontend/src/views/HomeView.vue

```diff
@@ -8,39 +8,67 @@ import { ref } from 'vue'
 import Checkbox from 'primevue/checkbox'
 import Navbar, { type MenuOption } from '@/components/TopNav.vue'
 import BaseIcon from '@/icons/BaseIcon.vue'
+import { useAuthStore } from '@/stores/auth'
+import { useRouter } from 'vue-router';
+
+const authStore = useAuthStore()
+
+const signedIn = authStore.signedIn
+console.log('signed in?: ', signedIn);
+const name = signedIn ? authStore.name : ''
+console.log('username: ', name);
 
 const topNavActions: MenuOption[] = [
-  {
+
+  !signedIn && {
     labelKey: 'sign_up',
     to: '/sign-up'
   },
```

Comment:
```suggestion
  ...(signedIn
    ? [
        {
          labelKey: 'sign_up',
          to: '/sign-up'
        }
      ]
    : []),
```

This results in no type error, but does the same.